Compile error

**Problem description**
Building shared directly results in the following error:
```
error[E0432]: unresolved imports `tokio_rustls::rustls::ServerCertVerified`, `tokio_rustls::rustls::ServerCertVerifier`
  --> shared/src/lib.rs:50:43
   |
50 | use tokio_rustls::rustls::{RootCertStore, ServerCertVerified, ServerCertVerifier, TLSError};
   |                                           ^^^^^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^^^^^ no `ServerCertVerifier` in the root
   |                                           |
   |                                           no `ServerCertVerified` in the root

For more information about this error, try `rustc --explain E0432`.

```

**Problem location**
This appears to be because when not build through the root toml file the `dangerous_configuration` feature from `tokio-rustls` is not enabled causing these classes to be missing.

**Suggestions for an improvement**
Specify the required feature in shared/Cargo.toml

--- a/shared/Cargo.toml
+++ b/shared/Cargo.toml
@@ -29,4 +29,4 @@ num-traits = "0.2"
 num-derive = "0.3"
 regex = "1.3"
 lazy_static = "1.4"
-tokio-rustls = "0.22.0"
+tokio-rustls = { version = "0.22.0", features = ["dangerous_configuration"] }

